<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>421</height>
    </rect>
   </property>
   <widget class="QWidget" name="Server">
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>478</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>启动</string>
     </property>
    </widget>
    <widget class="QLabel" name="serverPort">
     <property name="geometry">
      <rect>
       <x>328</x>
       <y>20</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>端口：</string>
     </property>
    </widget>
    <widget class="QLabel" name="serverIP">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>服务器IP：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="userManageButton">
     <property name="geometry">
      <rect>
       <x>588</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>用户管理</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="serverIPEdit">
     <property name="geometry">
      <rect>
       <x>108</x>
       <y>20</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="messageHistoryBox">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>60</y>
       <width>351</width>
       <height>331</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>消息历史</string>
     </property>
     <widget class="QTextEdit" name="messageHistoryEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>351</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="serverPortEdit">
     <property name="geometry">
      <rect>
       <x>368</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="userListBox">
     <property name="geometry">
      <rect>
       <x>388</x>
       <y>60</y>
       <width>301</width>
       <height>331</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>用户列表</string>
     </property>
     <widget class="QTableWidget" name="userListWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>301</width>
        <height>311</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>用户名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>状态</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IP</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="userManage"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
